aiida_pseudo/groups/family/sssp.py

```diff
@@ -50,15 +53,17 @@ def format_configuration_label(cls, configuration: SsspConfiguration) -> str:
         )
 
     @classmethod
-    def format_configuration_filename(cls, configuration: SsspConfiguration, extension: str) -> str:
+    def format_configuration_filename(cls, configuration: SsspConfiguration, extension: str, patch_version: Optional[str] = None) -> str:
```

Comment:
Not sure if it is a good idea to keep the patch version as a separate argument. Shouldn't we try to integrate this in the `SsspConfiguration`. It already contains a `version` attribute. If that is `1.2.3` than the patch version is `3` by definition. So to have a separate argument that can cause inconsistencies seems weird.

Reply:
> Thanks for the review, @sphuber! I made some first corrections, but have a look at [this comment](https://github.com/aiidateam/aiida-pseudo/pull/104#discussion_r686941153) and see if you agree with my reasoning here before I continue.

I can remember the discussion about not fixing the patch number explicitly as that would require updates of the plugin if a new patch is released of the family, but I am having difficulty seeing how best to integrate this limitation. Can you share the content of the YAML mapping? Than I can form a better idea of what to do

Reply:
Sure! It's a very simple mapping of the minor version to the latest patch version:

```yaml
---
# This maps each minor version to the latest patch version
# corresponding to that minor version
'1.0': '1.0'
'1.1': '1.1.2'
```